torch/csrc/jit/script/python_sugared_value.cpp

```diff
@@ -239,6 +239,10 @@ std::shared_ptr<SugaredValue> OverloadedMethodValue::call(
                          << err.str();
 }
 
+Value* ModuleValue::asValue(const SourceRange& loc, Function& m) {
```

Comment:
@zdevito do you think this is safe to add now that modules are first-class?

Reply:
Yes, it should be there. I am surprised that stuff worked without it actually.